pyHS100/cli.py

```diff
@@ -129,6 +129,15 @@ def state(ctx, dev):
 
     click.echo(click.style("Device state: %s" % "ON" if dev.is_on else "OFF",
                            fg="green" if dev.is_on else "red"))
+    if dev.num_children > 0:
+        is_on = dev.is_on()
+        aliases = dev.get_alias()
+        for child in dev.num_children:
```

Comment:
This line raises an error for me:
```
File "pyHS100/cli.py", line 135, in state                                                                                                                                                                                                                       for child in dev.num_children:                                                                                                                                                                                                                                                                                           TypeError: 'int' object is not iterable
```

Reply:
Worked after changing the line to:
```Python
for child in range(dev.num_children):
```

Reply:
Indeed, thanks for testing and spotting this! Does the output otherwise look ok?